src/st_stuff.c

```diff
@@ -851,6 +851,14 @@ void ST_drawWidgets(boolean refresh)
   int i;
   int maxammo = plyr->maxammo[weaponinfo[w_ready.data].ammo];
 
+  if (!crispy_hud)
+  {
+    int x = WIDESCREENDELTA;
```

Comment:
Could you please add a comment here about which areas are copied and why?

Reply:
I suggest copying the entire STBAR and removing the `oldinum` optimization from `STlib_updateMultIcon`. I'm sure there won't be a measurable difference in performance, and who knows what kind of STBAR will be invented in the future.

Reply:
So, if you could show me an elegant solution that works with all current HUDs, that'd be convincing. 😉

Reply:
Not sure if it's elegant, but it's simpler 😄